lib/galaxy/tool_util/parser/xml.py

```diff
@@ -95,6 +95,12 @@ def parse_edam_topics(self):
             return []
         return [edam_topic.text for edam_topic in edam_topics.findall("edam_topic")]
 
+    def parse_xrefs(self):
+        xrefs = self.root.find("xrefs")
+        if xrefs is None:
+            return []
+        return [xref.text for xref in xrefs.findall("xref") if xref.get("type") in {'bio.tools'}]
```

Comment:
```suggestion
        return [dict(content=xref.text.strip(), reftype=xref_elem.attrib['type']) for xref in xrefs.findall("xref") if xref.get("type")]
```

Here I have used `content` and `reftype` as keys, but maybe `type` and `value` (which you used in the yaml test) may make more sense and be more consistent. The Vue file would also need to use whatever keys you decide to use.

Reply:
Finally I choosed `reftype` and `value`